### ISSUE TITLE
MultiSelect a11y issues

### ISSUE THREAD

**Issue by nicoxxie** [carbon-design-system/carbon-components-react#1611]
## Detailed description

> Describe in detail the issue you're having.
When I tab onto a MultiSelect component, the focus is correct but cannot do anything by keyboard.
![screen shot 2018-12-05 at 2 06 01 pm](https://user-images.githubusercontent.com/17915105/49540951-f987a780-f896-11e8-95ac-5f155a47e895.png)

Then I tab again, the focus is on the dropdown arrow, I can use the keyboard to expand it but the focus is not shown at all.
![screen shot 2018-12-05 at 2 06 20 pm](https://user-images.githubusercontent.com/17915105/49540950-f987a780-f896-11e8-820b-a5bcc28ef15a.png)

> Is this issue related to a specific component?
MultiSelect

> What did you expect to happen? What happened instead? What would you like to see changed?
Hover on the first tab and able to expand by keyboard on there.

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?
6.55.5

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Cloud

## Steps to reproduce the issue
Can be tested on http://react.carbondesignsystem.com/?selectedKind=MultiSelect&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by frastlin** [comment]
Confirm using NVDA 2018.3.2 on Chrome Version 71.0.3578.80 (Official Build) (64-bit) and 63.0.3 (64-bit).

I would like to add that the element that is used to represent the combobox is not an element I would click on and is not labeled with aria role="combobox". A better element would be a `<input type="text" role="combobox">`. For a perfect implementation of the desired behavior, please see the [AccDC Example](http://whatsock.com/tsg/Coding%20Arena/ARIA%20Comboboxes/ARIA%20Comboboxes%20(Simulated,%20Readonly%20Multiselect)/demo.htm)

**Comment by yacineMTB** [comment]
https://github.com/IBM/carbon-components-react/pull/1731

**Comment by shinytoyrobots** [comment]
Closed by #1731 merge.

**Comment by nicoxxie** [comment]
Tried tabbing again, same issue still happen, shows the focus but does not open the component with press enter/down/space .
I manually click opened it but failed at try to tab into the check boxes.

![Mar-21-2019 10-28-58](https://user-images.githubusercontent.com/17915105/54763588-3d8f8580-4bc4-11e9-8294-8c57f08fe150.gif)


@shinytoyrobots can you reopen?

**Comment by icemanlily** [comment]
@shinytoyrobots Could you re-open this issue?  It is not fixed.

**Comment by shinytoyrobots** [comment]
@joshblack, when available could you and @yacineMTB discuss this issue - the fix in #1731 PR has apparently not resolved the issue?

**Comment by johnsteinbacher** [comment]
@shinytoyrobots - would you please have someone add an ETA to resolving this issue.  This is holing up AVT exit.  If you need help validating a fix please reach out to @nicoxxie .

**Comment by emyarod** [comment]
@johnsteinbacher @nicoxxie does #2239 (currently still in progress) address the issues you are facing?

**Comment by dakahn** [comment]
@nicoxxie looks like we're teeing up a fix -- just wanted to check with you that #2239 applies here.

**Comment by emyarod** [comment]
@johnsteinbacher @nicoxxie closing as #2239 has been merged. feel free to reopen this issue if it persists

**Comment by nicoxxie** [comment]
@dakahn @emyarod Sorry I was on vacation, unfortunately we have not migrated to carbon v10 yet... and I see this still happens on the latest v6 carbon react, will see if the update happens soon.

**Comment by emyarod** [comment]
oh I see, I overlooked the v6 part. will need to backport my v10 fix
